string waste
geometry_msgs/Point center